samples/managed-app-sample/createUiDefinition.json

```diff
@@ -0,0 +1,189 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json#",
+	"handler": "Microsoft.Azure.CreateUIDef",
```

Comment:
To improve discoverability, can you please include this sample in the table in repo's readme.md https://github.com/Azure-Samples/kubernetes-offer-samples#samples-in-this-repo ?